src/central-server/openapi-model/src/main/resources/openapi-definition.yaml

```diff
@@ -4826,6 +4826,68 @@ components:
         - member_class
         - member_code
         - server_code
+    ManagementRequestView:
```

Comment:
maybe ManagementRequestDetails would be more correct name? Then just extend ManagementRequest object and add missing data (probably certificates?)